packages/orchestration/src/orchestration-api.ts

```diff
@@ -74,7 +75,9 @@ export type OrchestrationAccount<CI extends ChainInfo> =
     (CI extends CosmosChainInfo
       ? CI['chainId'] extends `agoric${string}`
         ? LocalAccountMethods
-        : CosmosChainAccountMethods<CI>
+        : CI['chainId'] extends `noble${string}`
+          ? CosmosChainAccountMethods<CI> & NobleMethods
```

Comment:
see https://github.com/Agoric/agoric-sdk/pull/10967/files#r1946946855

Reply:
seems like we could use the presence of `cctpDestinationDomain`.